### DISCUSSION TITLE
Add a root directory for Render Plugin

### DISCUSSION THREAD

**Discussion by TigersWay** [11ty/eleventy#4114] [Enhancement Queue]
**Is your feature request related to a problem? Please describe.**
I do not like the fact that templates calling "micro-template(s)" have to be aware of the structure of the project. Right now they have to know where are, from the root of the project, theses rendered files.

**Describe the solution you'd like**
I would love to be able to pass the "root path" when initializing this plugin.
.eleventy.js
```
    eleventyConfig.addPlugin(EleventyRenderPlugin, {rootPath: "/site/_themes/grayTheme"})
```
any template
```
    {% renderFile "./_includes/blocks/blogpost.md" %}
```
blogpost.md is in `/site/_themes/grayTheme/_includes/blocks`

**Describe alternatives you've considered**
So far - I am a nunjucks only kinda guy - I can render file(s) from a pre-called nunjucks environment.

**Additional context**
As you can see in the example I gave before I often have the includes & layouts folders under some theme logic like: `/site/_themes/grayTheme/_includes` for example. I have that "logic" inside the config file of course, but don't want to carry it around in templates.

Thanks

**Comment by ivanminutillo** [comment]
it seems not possible to load external file with renderFile like :
``` {% renderFile "https://raw.githubusercontent.com/bonfire-networks/blog/master/code_of_conduct.md" %}```
Any idea how to do it?

**Comment by zachleat** [comment]
@ivanminutillo please file a separate issue!

**Comment by zachleat** [comment]
Cross-linking to #4101 and https://www.11ty.dev/docs/plugins/render/
